Readability.js

```diff
@@ -230,7 +230,7 @@ Readability.prototype = {
     if (this._docJSDOMParser && nodeList._isLiveNodeList) {
       throw new Error("Do not pass live node lists to _replaceNodeTags");
     }
-    for (var i = nodeList.length - 1; i >= 0; i--) {
+    for (var i = 0; i < nodeList.length; i++) {
```

Comment:
The index doesn't seem to be used for anything else than the iteration itself, you should be able to use `for (const i of nodeList) {/* ... */}` here.

See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of.